app/lib/methods/search.ts

```diff
@@ -38,7 +38,8 @@ export const localSearchSubscription = async ({ text = '', filterUsers = true, f
 		t: item.t,
 		encrypted: item.encrypted,
 		lastMessage: item.lastMessage,
-		status: item.status
+		status: item.status,
+		teamMain: item.teamMain
```

Comment:
During the QA, the QA team found that during the search, the team icon isn't rendering properly, and is the same problem as the share view, we missed to pass the param teamMain
#### Before
![Screen Shot 2023-01-24 at 19 02 34](https://user-images.githubusercontent.com/47038980/214431053-aec84115-4691-475d-8c31-00dea457c742.png)

#### After
![Screen Shot 2023-01-24 at 19 01 50](https://user-images.githubusercontent.com/47038980/214430926-66169fa6-5a36-4737-9d42-1d692a0ac277.png)